src/index.js

```diff
@@ -232,6 +235,7 @@ export default class VueI18n {
 
   get fallbackLocale (): Locale { return this._vm.fallbackLocale }
   set fallbackLocale (locale: Locale): void {
+    this._vm.$set(this._vm, '_localeChainCache', new Map())
```

Comment:
I think that `_localeChainCache` don't need set Vue instance, because it's not used reactive by user.

Reply:
```suggestion
    this._localeChainCache = new Map()
```